mpc_follower_wrapper/launch/mpc_follower_wrapper.launch

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<launch>
+    <remap from="final_waypoints" to="/guidance/carma_final_waypoints"/>
+    <!-- Pure Pursuit Node -->
+    <group>
+        <!--<remap from="vehicle_info/wheel_base" to="/vehicle_wheel_base"/>-->
```

Comment:
I think this is required

Reply:
updated it with the mpc wheelbase parameter name. However, I tried it before during the integration test and did not update the parameter.